### DISCUSSION TITLE
Unreal Engine iOS Builds

### DISCUSSION THREAD

**Discussion by coopbri** [sickcodes/Docker-OSX#713] [Q&A]
Would it be practical and feasible to use this project to streamline CI for Unreal Engine iOS builds (relevant discussion [here](https://forums.unrealengine.com/t/ios-remote-build-services/115139/4)), or would it be better to use a VM (such as [this project](https://github.com/kholia/OSX-KVM))?

If so, I would love to hear people's experiences with this. I am particularly curious if headless builds (such as outlined [here](https://github.com/sickcodes/Docker-OSX#building-a-headless-container-from-a-custom-image)) can work well.

Similar discussion for Unity: https://github.com/sickcodes/Docker-OSX/discussions/180

**Comment by abhisheksagi** [comment]
I'm about to try this with Unreal Horde Setup. Can report back with results
